docs/ref/checks.txt

```diff
@@ -165,9 +165,11 @@ Model fields
 * **fields.E002**: Field names must not contain ``"__"``.
 * **fields.E003**: ``pk`` is a reserved word that cannot be used as a field
   name.
-* **fields.E004**: ``choices`` must be an iterable (e.g., a list or tuple).
-* **fields.E005**: ``choices`` must be an iterable containing ``(actual value,
-  human readable name)`` tuples.
+* **fields.E004**: ``choices`` must be a dictionary or an iterable
```

Comment:
I believe this needs updating to match the error message from `model/fields/__init__.py`:

```
ticket_31262.Student.year_in_school: (fields.E004) 'choices' must be a mapping (e.g. a dictionary) or an iterable (e.g., a list or tuple).
```